web/html/src/manager/admin/config/monitoring-admin.js

```diff
@@ -184,9 +245,18 @@ const MonitoringAdmin = (props) => {
               </div>
               <div className="col-md-8">
                 { exportersStatus ? 
-                  <ExportersList exporters={exportersStatus}/> :
+                  <ExportersList exporters={exportersStatus} messages={exportersMessages}/> :
                   <ListPlaceholder/>
-                  }
+                }
+                {
+                  restartNeeded ? 
+                    <div><Icon type="system-warn" className="fa-1-5x"/>
```

Comment:
If this is all about `restarting`, why not using a `fa fa-refresh text-warning` icon?